CITATION.cff

```diff
@@ -0,0 +1,32 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.2.0
+message: If you use this software, please cite the following article and the appropriate version of the archived data.
+title: "Matbench: Benchmarks for materials science property prediction"
+authors:
+- given-names: Alex
+  family-names: Dunn
+  affiliation: UC Berkeley
+  orcid: https://orcid.org/0000-0002-8567-1879
+repository-code: https://github.com/materialsproject/matbench
+url: https://matbench.materialsproject.org/
+license: MIT
+preferred-citation:
```

Comment:
This "preferred-citation" field is used by GitHub to populate the "Cite this repository" button in the repo.